src/sentry/incidents/models/alert_rule.py

```diff
@@ -164,14 +164,14 @@ def conditionally_subscribe_project_to_alert_rules(
                             "condition": activation_condition,
                         },
                     )
-                    # attempt to subscribe the alert rule
                     created_subscriptions.extend(
                         alert_rule.subscribe_projects(
                             projects=[project],
                             monitor_type=AlertRuleMonitorType.ACTIVATED,
                             query_extra=query_extra,
                             activation_condition=activation_condition,
                             activator=activator,
+                            timebox=True,
```

Comment:
activated alerts are timeboxed from `QuerySubscription` creation till end of the `SnubaQuery.time_window`

Reply:
Rather than creating a boolean and passing it around, could we set a property on the model which checks to see if there's a time_window + time_start associated with it?